File: src/main/java/ru/fullrest/mfr/plugins_configuration_utility/model/repository/GroupRepository.java
package ru.fullrest.mfr.plugins_configuration_utility.model.repository;

import org.springframework.data.repository.CrudRepository;
import ru.fullrest.mfr.plugins_configuration_utility.model.entity.Group;

import java.util.List;

/**
 * Created on 02.11.2018
 *
 * @author Alexey Plekhanov
 */
public interface GroupRepository extends CrudRepository<Group, Integer> {
    List<Group> findAllByActiveIsTrue();
}

File: desktop/application/src/main/kotlin/ru/fullrest/mfr/plugins_configuration_utility/model/repository/GroupRepository.kt
package ru.fullrest.mfr.plugins_configuration_utility.model.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.Query
import ru.fullrest.mfr.plugins_configuration_utility.model.entity.Group

interface GroupRepository : JpaRepository<Group, Int> {
    fun findAllByOrderByValue(): List<Group>

    @Query(
        """SELECT g FROM Group g 
JOIN FETCH g.releases 
ORDER BY g.value"""
    )
    fun findAllWithReleases(): List<Group>
}